src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.Tool.csproj

```diff
@@ -28,23 +28,18 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="..\..\build\docker\linux-build.dockerfile" Link="linux-build.dockerfile" />
-    <None Include="Datadog.Trace.Tools.Runner.proj" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.8.0" />
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="home\**\*.*" Pack="true" PackagePath="\home">
+    <Content Include="..\Datadog.Trace.AutoInstrumentation\home\**\*.*" Pack="true" PackagePath="\home" LinkBase="home">
```

Comment:
Have we tested the output of the changes to the two dd-trace runners? Currently we don't test this output in the CI pipeline at all (we really should!) so we can end up breaking it without noticing until _after_ the next release

Reply:
Manually inspecting the output, the binaries are different (because of different builds), but the list of files is identical. Is that good for now?

Reply:
I think so :) one quick check could be to install the `dd-trace` global tool, set the `--tracer-home` variable to the dd-tracer-home directory that's built, and confirm it can still trace a sample app?